### PR TITLE
Create `portProp` method 

### PR DESCRIPTION
**Pull request by vtalas**
- portProp allows to set port properties by port id 
- added `data-group` attribute to the port DOM element 

ready @kumilingus

### CODE REVIEW

File: src/ports.js [hunk 1]
```diff
@@ -239,6 +239,35 @@
...
+         */
+        portProp: function(portId, path, value, opt) {
+
+            var index = this.getPortIndex(portId);
+
+            if (index === -1) {
+                throw new Error('Element: unable to find port with id ' + portId);
+            }
+
+            if (!_.startsWith(path, '/')) {
```
Comment (kumilingus) [inline src/ports.js]: Note that if `path` starts with `/` it points to a different property in an object.

`'a/b'    { a: { b: 'value' }`

`'/a/b'   { '': { a: { b: 'value' }}`

Comment (vtalas) [inline src/ports.js]: in this case at least one `/` has to be there, as the `path` is just a part of the 'result' path, 
` var portPath = ['ports/items/', index, path].join('');`

Comment (kumilingus) [inline src/ports.js]: Sure. The `/` has to be there in any case. Even if the `path` already starts with `/`.

    var portPath = ['ports/items/', index, '/', path].join('');

File: src/ports.js [hunk 2]
```diff
@@ -239,6 +239,35 @@
...
+                path = '/' + path;
+            }
+
+            var portPath = ['ports/items/', index, path].join('');
+
+            if (_.isUndefined(value)) {
+                return this.prop(_.trimRight(portPath, '/'));
```
Comment (kumilingus) [inline src/ports.js]: The same applies here. You can not trim last `/`, as you would change the path meaning.

**Pull request merged**